<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreScreen</class>
 <widget class="QDialog" name="StoreScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>903</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tienda de Sudaderas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="cartbutton">
   <property name="geometry">
    <rect>
     <x>704</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #000000;
    border: 2px solid #000000;
    border-radius: 6px;
    padding: 6px 12px;
    font: 14pt &quot;Impact&quot;;
}

QPushButton:hover {
    background-color: #000000;
    color: #ffffff;
}

QPushButton:pressed {
    background-color: #222222;
    color: #ffffff;
}

QPushButton:disabled {
    background-color: #eeeeee;
    color: #999999;
    border-color: #cccccc;
}
</string>
   </property>
   <property name="text">
    <string>ðŸ›’ Carrito</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 48pt &quot;Akira Expanded&quot;;</string>
   </property>
   <property name="text">
    <string>SUDADERAS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sweater0.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sweater1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sweater2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>510</y>
     <width>291</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sweater3.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addbutton2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #000000;
    color: #888888;
    border: none;
    border-radius: 6px;
    padding: 6px 12px;
    font: 14pt &quot;Impact&quot;;
}

QPushButton:hover {
    color: #ffffff;
    font: 14pt &quot;Impact&quot;;
}

QPushButton:pressed {
    background-color: #111111;
    font: 14pt &quot;Impact&quot;;
}

QPushButton:disabled {
    background-color: #222222;
    color: #555555;
    font: 14pt &quot;Impact&quot;;
}
</string>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addbutton3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>460</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #000000;
    color: #888888;
    border: none;
    border-radius: 6px;
    padding: 6px 12px;
    font: 14pt &quot;Impact&quot;;
}

QPushButton:hover {
    color: #ffffff;
    font: 14pt &quot;Impact&quot;;
}

QPushButton:pressed {
    background-color: #111111;
    font: 14pt &quot;Impact&quot;;
}

QPushButton:disabled {
    background-color: #222222;
    color: #555555;
    font: 14pt &quot;Impact&quot;;
}
</string>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addbutton4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>830</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #000000;
    color: #888888;
    border: none;
    border-radius: 6px;
    padding: 6px 12px;
    font: 14pt &quot;Impact&quot;;
}

QPushButton:hover {
    color: #ffffff;
    font: 14pt &quot;Impact&quot;;
}

QPushButton:pressed {
    background-color: #111111;
    font: 14pt &quot;Impact&quot;;
}

QPushButton:disabled {
    background-color: #222222;
    color: #555555;
    font: 14pt &quot;Impact&quot;;
}
</string>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addbutton1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>460</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #000000;
    color: #888888;
    border: none;
    border-radius: 6px;
    padding: 6px 12px;
    font: 14pt &quot;Impact&quot;;
}

QPushButton:hover {
    color: #ffffff;
    font: 14pt &quot;Impact&quot;;
}

QPushButton:pressed {
    background-color: #111111;
    font: 14pt &quot;Impact&quot;;
}

QPushButton:disabled {
    background-color: #222222;
    color: #555555;
    font: 14pt &quot;Impact&quot;;
}
</string>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #f77f7f;
	
	font: 87 12pt &quot;Segoe UI Black&quot;;
}
</string>
   </property>
   <property name="text">
    <string>$750.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Sudadera Oversize con GrÃ¡ficos Grandes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #f77f7f;
	
	font: 87 12pt &quot;Segoe UI Black&quot;;
}
</string>
   </property>
   <property name="text">
    <string>$1100.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Sudadera con Parche Bordado + Capucha Forrada</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>430</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #f77f7f;
	
	font: 87 12pt &quot;Segoe UI Black&quot;;
}
</string>
   </property>
   <property name="text">
    <string>$950.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>380</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Sudadera Tie-Dye con Zipper Lateral</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>780</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Sudadera Minimalista con Toques Techwear</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>830</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #f77f7f;
	
	font: 87 12pt &quot;Segoe UI Black&quot;;
}
</string>
   </property>
   <property name="text">
    <string>$1400.00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closesession">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>850</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #000000;
    border: 2px solid #000000;
    border-radius: 6px;
    padding: 6px 12px;
    font: 14pt &quot;Impact&quot;;
}

QPushButton:hover {
    background-color: #000000;
    color: #ffffff;
}

QPushButton:pressed {
    background-color: #222222;
    color: #ffffff;
}

QPushButton:disabled {
    background-color: #eeeeee;
    color: #999999;
    border-color: #cccccc;
}
</string>
   </property>
   <property name="text">
    <string>Cerrar SesiÃ³n</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
